### DISCUSSION TITLE
How to change default text in Select component.

### DISCUSSION THREAD

**Discussion by MonkaMaczna** [JedWatson/react-select#5019] [Q&A] [answered]
So, by default there is default String in the options. Is it possible to change that from "Select..." to for example same thing in another language?

![image](https://user-images.githubusercontent.com/70438268/150384454-6f9a3362-1e5c-42e1-bc4b-d2526341accb.png)

**Comment by Rall3n** [comment, marked as answer]
Hello @MonkaMaczna,

you can customize this using the `placeholder` prop.

```jsx
<Select
  placeholder="My custom placeholder"
/>
```
![image](https://user-images.githubusercontent.com/15950630/150416656-59406b23-0e59-45a6-9914-e71d3c08768e.png)

**Reply by MonkaMaczna** [reply]
Thank you so much!
